README.md

```diff
@@ -0,0 +1,51 @@
+# piControl
+
+piControl is a kernel module for interfacing with RevPi hardware. It provides a common interface to all RevPi related IOs, which a user can consume via `/dev/piControl0`.
+
+## Usage
+
+## Build the module
+
+All the following steps need to be executed on a RevPi device running the official image. It is also possible to compile the module on another system, but the instructions may vary.
+
+Install kernel headers:
+
+```
+sudo apt update
+sudo apt install raspberrypi-kernel-headers
+```
+
+Checkout repository and switch working directory:
+
+```
+git clone https://github.com/RevolutionPi/piControl.git
+cd piControl
+```
+
+Compile module sources for current kernel and architecture:
+
+```
+KDIR=/usr/src/linux-headers-$(uname -r)/ make
+```
+
+## Load the compiled module
+
+Before the newly module can be loaded, the currently module needs to unloaded:
```

Comment:
newly **compiled** module ... currently **loaded** module ... needs to **be** unloaded